core/domain/feedback_jobs_continuous.py

```diff
@@ -179,10 +201,27 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Map function.
+
+        Args:
+            item: A single element of the type given by entity_class().
```

Comment:
This could be more specific:

    item: FeedbackThreadModel. A feedback thread model instance.